### ISSUE TITLE
Tag "addr:street" not matching a nearby street name nor alt_name

### ISSUE THREAD

**Issue by grhoten** [osm-fr/osmose-backend#2066]
This issue is related to [analyser_osmosis_relation_associatedStreet.py#L652](https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_osmosis_relation_associatedStreet.py#L652)

The problem is that it doesn’t take alt_name into account. It’s not uncommon that a street can have an optional direction in the street name. Such information can be located in the alt_name value. The street signs may omit the direction, but property tax public records can include the directionality.

Here’s an example building that has an address that doesn’t match the nearby street, but the street already has the correct matching name in the alt_name value.
https://www.openstreetmap.org/way/39129357

Here is a link to one of the numerous issues for the length of the street. https://osmose.openstreetmap.fr/en/issue/ef94abd6-cd46-89bd-b08e-d1fb720275d7

For this check, please check the value of both name and alt_name for the street name.

**Comment by frodrigo** [comment]
See #1012

**Comment by grhoten** [comment]
Yes, this request is very similar to the requests seen in the following issues.

https://github.com/osm-fr/osmose-backend/issues/1012
https://github.com/osm-fr/osmose-backend/issues/564
https://github.com/osm-fr/osmose-backend/issues/551

**Comment by grhoten** [comment]
If you need numerous examples in one county where a large number of them are valid due to the optional direction being included in the name, here is an example. https://osmose.openstreetmap.fr/en/issues/open?source=39470&item=2060&class=19

Some of the issues in that list are valid, but a lot of them are not valid issues.
